apps/nsq_to_file/nsq_to_file.go

```diff
@@ -34,6 +34,7 @@ var (
 
 	rotateSize     = flag.Int64("rotate-size", 0, "rotate the file when it grows bigger than `rotate-size` bytes")
 	rotateInterval = flag.Duration("rotate-interval", 0*time.Second, "rotate the file every duration")
+	rotateCheckInterval = flag.Duration("rotate-check-interval", 30*time.Second, "check whether the file should be rotated every duration")
```

Comment:
I think name and description is not really right - the check whether the file should be rotated is done on every single message, this really just affects how often the files (including empty files I guess) are synced to disk

Reply:
Hmmmm, what do you have in mind?

Reply:
Yea, I agree, `--sync-interval` makes more sense. I can whip that up or do you wanna do the honors @alpaker?

Reply:
Sure.  Regarding the description: If we're going to foreground the flush interval as what's controlled by this option, then I suggest omitting mention of rotation altogether.  That the need for rotation is checked on msg receipt isn't explicitly documented, which makes the significance of "check if file should should have been rotated since the last message received" unclear.  How about: 
```
syncInterval = flag.Duration("sync-interval", 30*time.Second, "flush messages to disk every duration")
```

Reply:
Sounds good, except `s/flush/sync` to match the flag name?

Reply:
all looks good to me, I was just trying to appease the reasoning behind the original naming :)